File: 2_fibonacci/fibonacci.py
# coding: utf8
# Python implementation of the Fibonacci Series
# Author: Eric Alcaide

def fibonacci(a):
    prev, actual = 0, 1
    
    for q in range(a):
        actual, prev = actual+prev, actual
        print(actual)

def main():
	print("Enter the number upto which to take fibonacci:",end=" ")
	a=int(input())
	fibonacci(a)

if __name__ == "__main__":
    main()

File: 2_fibonacci/fibonacci.c
/* Fibonacci implementation for Rosetta Stone project */

#include <stdio.h>

void fibonacci()
{
    int prev = 0;
    int cur = 1;
    int n_cur;
    int i;
    for(i = 0; i < 10; i++) {
        n_cur = cur + prev;
        prev = cur;
        cur = n_cur;
        printf("%d ", cur);
    }
    printf("\n");
}

int main(void)
{
    fibonacci();
}
